packages/opencensus-core/test/test-tracer.ts

```diff
@@ -354,12 +355,12 @@ describe('Tracer', () => {
     it('should support nested children', () => {
       tracer.startRootSpan(options, (rootSpan) => {
         assert.strictEqual(rootSpan.numberOfChildren, 0);
-        const child1 =
-            tracer.startChildSpan('child1', types.SpanKind.UNSPECIFIED);
+        const child1 = tracer.startChildSpan(
+            {name: 'child1', kind: types.SpanKind.UNSPECIFIED});
```

Comment:
To simplify these tests and the API, can `kind` just have a default value?  Then we wouldn't need to specify it again and again.